_includes/mailchimp2.html

```diff
@@ -0,0 +1,78 @@
+<!-- Begin Mailchimp Signup Form -->
+<div id="mc_embed_signup">
+  <form
+    action="https://smartlogic.us19.list-manage.com/subscribe/post?u=3df423f51248435e7c3bf413b&amp;id=614560afea"
+    method="post"
+    id="mc-embedded-subscribe-form"
+    name="mc-embedded-subscribe-form"
+    class="validate my-5"
+    target="_blank"
+    novalidate
+  >
+    <p><strong>Are your ready to start building?</strong></p>
+    <div id="mc_embed_signup_scroll" class="d-flex">
+      <div class="mc-field-group w-100 mr-4">
+        <input
```

Comment:
On mobile this input field is too small. The pattern from connect4bot in mailchimp3.html works better where the button flows to the next line.